backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/utils/MDevTypesUtils.java

```diff
@@ -1,48 +1,81 @@
 package org.ovirt.engine.core.common.utils;
 
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 import org.ovirt.engine.core.common.businessentities.VM;
+import org.ovirt.engine.core.common.businessentities.VmDevice;
+import org.ovirt.engine.core.common.businessentities.VmDeviceGeneralType;
 import org.ovirt.engine.core.common.utils.customprop.SimpleCustomPropertiesUtil;
+import org.ovirt.engine.core.compat.Guid;
 import org.ovirt.engine.core.compat.Version;
 
 public class MDevTypesUtils {
 
+    public static final String DEVICE_ID = "deviceId";
```

Comment:
What information will DEVICE_ID and MDEV_TYPE? I mean, which one would keep the 'nvidia_id' reference?

Reply:
DEVICE_ID contains the device Guid, MDEV_TYPE contains the mdev type (e.g. nvidia-22).